conf/application.conf

```diff
@@ -90,7 +90,7 @@ platform.frontend.host = "http://localhost:9250"
 
 play.i18n.langs = ["en", "cy"]
 
-# Change this value to true to enable Welsh translations to be loaded from messages.cy
 features.welsh-language-support = true
```

Comment:
So minor, but maybe just an line break here as it looks like both need to enabled to support Welsh...

Reply:
It looks like I deleted the comment for some reason, adding back now.